src/brian.py

```diff
@@ -236,12 +238,10 @@ def start_new():
         ):
 
             # Get new sequence metadata from radar control
-            reply = so.recv_obj(
+            sigp = so.recv_pyobj(
```

Comment:
I would like to add in verification that the received object is an instance of the expected class from message_formats.py. I think you already did that, it's just not used in this case?